UPGRADING.md

```diff
@@ -0,0 +1,28 @@
+# Upgrading from SitePrism 2.x to 3.x
+
+## Default Load Validations
+
+Site Prism 2 Used to contain 1 inbuilt load validation for all Pages that inherited from `SitePrism::Page`
+This has now been removed. If you wish to retain the previous functionality, add the following load validation
```

Comment:
to all the direct descendant of `SitePrism::Page`.
```ruby
class MyPage < SitePrism::Page
```
become
```ruby
class MyPage < SitePrism::Page
   load_validation [displayed?, "Expected #{current_url} to match #{url_matcher} but it did not."]
end
```
or 
```ruby
class BasePage < SitePrism::Page
   load_validation [displayed?, "Expected #{current_url} to match #{url_matcher} but it did not."]
end
class MyPage < BasePage
   load_validation [displayed?, "Expected #{current_url} to match #{url_matcher} but it did not."]
end
```

Reply:
yeah adding a fuller example is probably a good idea here.  It'll probably save at least a couple of questions.